src/webgpu/api/operation/rendering/blending.spec.ts

```diff
@@ -121,33 +125,33 @@ g.test('GPUBlendComponent')
     - component= {color, alpha} - whether to test blending the color or the alpha component.
     - srcFactor= {...all GPUBlendFactors}
     - dstFactor= {...all GPUBlendFactors}
-    - operation= {...all GPUBlendOperations}
-  `)
-  .cases(
```

Comment:
Maybe this change won't be necessary once we don't collect ALL stdout into one big file in our infra.